dynamic-demo-plugin/package.json

```diff
@@ -18,6 +18,7 @@
     "@patternfly/react-core": "4.162.2",
     "@patternfly/react-table": "4.31.7",
     "@types/react": "16.8.13",
+    "@types/react-measure": "^2.0.6",
```

Comment:
I understand that this is needed in our main package.json, but is this really required in the dynamic-demo-plugin? We don't use or export react-measure here.

Reply:
@jerolimov it is required based off the output of the build. According to the build output (build.sh)

`✨  Done in 330.80s.
~/src/github.com/openshift/console
~/src/github.com/openshift/console/dynamic-demo-plugin ~/src/github.com/openshift/console
yarn install v1.22.15
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "@openshift-console/dynamic-plugin-sdk > @patternfly/quickstarts@2.0.1" has unmet peer dependency "showdown@>=1.8.6".
warning " > react-i18next@11.12.0" has unmet peer dependency "i18next@>= 19.0.0".
warning " > file-loader@1.1.11" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
✨  Done in 12.46s.
yarn run v1.22.15
$ yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack
$ rm -rf dist
$ ts-node -O '{"module":"commonjs"}' node_modules/.bin/webpack
assets by status 87.3 KiB [cached] 22 assets
orphan modules 1.79 MiB [orphan] 1735 modules
runtime modules 18.3 KiB 12 modules
built modules 169 KiB (javascript) 294 bytes (consume-shared) [built]
  cacheable modules 169 KiB
    modules by path ./ 46.3 KiB
      modules by path ./components/ 36.4 KiB 12 modules
      modules by path ./utils/ 9.89 KiB 6 modules
    modules by path ../node_modules/ 123 KiB 7 modules
  consume-shared-module modules 294 bytes
    modules by path *.1 (singleton) 84 bytes 2 modules
    modules by path *.0 (singleton) 84 bytes 2 modules
    + 3 modules
  container entry 42 bytes [built] [code generated]

ERROR in /Users/jcaiani/src/github.com/openshift/console/dynamic-demo-plugin/tsconfig.json
/Users/jcaiani/src/github.com/openshift/console/dynamic-demo-plugin/tsconfig.json
[tsl] ERROR
      TS2688: Cannot find type definition file for 'react-measure'.
  The file is in the program because:
    Entry point for implicit type library 'react-measure'
ts-loader-default_9d8b55b1b989782b

webpack 5.73.0 compiled with 1 error in 9984 ms
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

Reply:
Interesting, thanks